packages/extension/app/client.js

```diff
@@ -1,4 +1,4 @@
-const { client, circularParser } = require('@packages/socket/lib/browser')
+const { client, circularParser } = require('@packages/socket/dist/lib/browser')
```

Comment:
I think once we upgrade to socket.io 3.x we do not need circularParser. It looks like patch-package is used instead. Also, looking at feat/evergreen, circular parser is not used anymore.